src/main/kotlin/com/kitchenforce/domain/orders/OrderTableRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.kitchenforce.domain.orders
+
+import org.springframework.data.repository.CrudRepository
+
+interface OrderTableRepository: CrudRepository<OrderTable, Long> {
```

Comment:
```CrudRepository```의 서브 인터페이스로 ```JpaRepository```를 사용하신다면 Paging 등등 부가적인 기능을 활용할 수 있기에 JpaRepository를 사용하는 것을 권장드립니다.
```suggestion
import org.springframework.data.repository.JpaRepository

interface OrderTableRepository: JpaRepository<OrderTable, Long> {
```